build/server/konnectors/digiposte.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
```

Comment:
I let you finish the review and will do a new pullrequest without the built files. Please don't merge it.